library/Vanilla/Formatting/Html/HtmlPlainTextConverter.php

```diff
@@ -0,0 +1,202 @@
+<?php
+/**
+ * @author Adam Charron <adam.c@vanillaforums.com>
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace Vanilla\Formatting\Html;
+
+use Garden\StaticCacheTranslationTrait;
+
+/**
+ * Class for converting HTML into plain text.
+ */
+class HtmlPlainTextConverter {
```

Comment:
That would be good to provide dedicated unit test for this class.
It looks to me as very specific for some context but class name sounds very general.

Reply:
Tests are provided for HTML plaintext conversion through the `output.txt` fixtures in BBCode, HTML, WYSIWYG, and Markdown formats.